doc/sphinxdoc/installing.rst

```diff
@@ -244,6 +244,11 @@ Building Essentia on iOS
 
 A lightweight version of Essentia can be `cross-compiled for iOS <FAQ.html#cross-compiling-for-ios>`_ from Mac OSX.
 
+Building Essentia for Web using asm.js or WebAssembly
+------------------------
```

Comment:
Continue the dash line to occupy the same length as a header for readability of the `rst` file.